lib/stateView/contract/removeContractDatabaseFactory.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * @param {createSVDocumentMongoDbRepository} createSVDocumentRepository
+ * @returns {removeContractDatabase}
+ */
+function removeContractDatabaseFactory(createSVDocumentRepository) {
+  /**
+   * Remove collection for each documentType in Contract
+   *
+   * @typedef {Promise} removeContractDatabase
+   * @param {SVContract} svContract
+   * @returns {Promise<*[]>}
+   */
+  async function removeContractDatabase(svContract) {
```

Comment:
We can drop the whole database instead of removing specific collections? I feel we don't need this logic if we commit state view transaction before we create databases.

Reply:
Nevermind :))